hikari/intents.py

```diff
@@ -219,13 +219,17 @@ class Intents(enums.Flag):
         This intent is privileged, and requires enabling/whitelisting to use.
     """
 
-    GUILD_BANS = 1 << 2
+    GUILD_MODERATION = 1 << 2
     """Subscribes to the events listed below.
 
+    * `GUILD_AUDIT_LOG_ENTRY_CREATE`
     * `GUILD_BAN_ADD`
     * `GUILD_BAN_REMOVE`
     """
 
+    GUILD_BANS = GUILD_MODERATION
+    """Deprecated alias of `GUILD_BANS`."""
```

Comment:
Can we do proper deprecation for this through a property?

Reply:
Would require making a whole descriptor class for it since can't mix classmethod and property on 3.8

Reply:
#1473 should aid with this :)